common/OpTestHMC.py

```diff
@@ -21,6 +21,16 @@
 #  This class can contain common functions which are useful for
 #  FSP_PHYP (HMC) platforms
 
+'''
+OpTestHMC
+---------
+
+HMC package contains all HMC related interfaces/functions.
+
+This package encapsulates all the functions which deals with the HMC in PowerVM
+systems
```

Comment:
This package encapsulates all HMC functions required to manage/control IBM Power server running PowerVM hypervisor.